PatientWebApplication/UserMicroserviceApi/Repository/GenericFileRepository.cs

```diff
@@ -0,0 +1,142 @@
+/***********************************************************************
+ * Module:  GenericFileRepository<TExtendsEntity>.cs
+ * Author:  Tamara
+ * Purpose: Definition of the Class Repository.GenericFileRepository<TExtendsEntity>
+ ***********************************************************************/
+
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UserMicroserviceApi.Model;
+
+namespace UserMicroserviceApi.Repository
+{
+    public class GenericFileRepository<T> where T : Entity
+    {
+        private string Path;
+
+        public GenericFileRepository() { }
+
+        public GenericFileRepository(string filePath)
+        {
+            Path = filePath;
+        }
+        public List<T> GetAll()
+        {
+            List<T> items = null;
+            try
+            {
+                using (var reader = new StreamReader(Path))
+                {
+                    string json = reader.ReadToEnd();
+                    items = JsonConvert.DeserializeObject<List<T>>(json);
+
+                }
+            }
+            catch (Exception ex)
+            {
+            }
+            return items;
+        }
+
+        public void New(T newEntity)
+        {
+            newEntity.id = generateid(newEntity);
+            List<T> list = GetAll();
+            list.Add(newEntity);
+            try
+            {
+                string JsonResult = JsonConvert.SerializeObject(list, Formatting.Indented);
+                string path = Path;
+                using (var writer = new StreamWriter(path))
+                {
+                    writer.Write(JsonResult);
+                }
```

Comment:
Write this in form:
"
using (var writer = new StreamWriter(Path))
                {
                    writer.Write(JsonConvert.SerializeObject(list, Formatting.Indented));
                }
"